addons/binding/org.openhab.binding.tplinksmarthome/ESH-INF/thing/HS103.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="tplinksmarthome"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="hs103">
+		<label>HS103</label>
+		<description>TP-Link HS103 Smart Wi-Fi Plug Mini</description>
```

Comment:
```suggestion
		<description>TP-Link HS103 Smart Wi-Fi Plug Lite</description>
```

Reply:
You're right. It says right on the box! I took the name off the Amazon web site, which shows it as "Mini" for some reason: https://www.amazon.com/TP-Link-HS105-KIT-Required-Assistant/dp/B06WD6Q8K9.

Reply:
That's because you are looking at the HS105 :wink:

Reply:
:P

Reply:
Done here: 
https://github.com/openhab/openhab2-addons/pull/5046/commits/f9d6090c494ce04bac9fec93cb571da928cd27d0

The irony is that even when I first ordered it, I assumed I had ordered the HS105. Only when trying to find them with OH did I realize they were these HS103.

Reply:
I had ordered 2 of the HS105. I just ordered two additional and I 2 did not catch the HS103 difference. I hope to use these with OH shortly. Please let me know how to install a binding in which these will work. 

Thanks.